src/Connector.js

```diff
@@ -1,14 +1,13 @@
-import _ from 'lodash';
-
 class Connector {
   constructor(client, token) {
     this.client = client;
     this.token = token;
 
-    this.onDataRequestedCb = _.noop();
-    this.onDataUpdatedCb = _.noop();
-    this.onDisconnectedCb = _.noop();
-    this.onReconnectedCb = _.noop();
+    const noop = () => undefined;
+    this.onDataRequestedCb = noop;
+    this.onDataUpdatedCb = noop;
+    this.onDisconnectedCb = noop;
+    this.onReconnectedCb = noop;
```

Comment:
I was looking through the [docs](https://lodash.com/docs/4.17.15#noop) and [source code](https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L15760)... this has the same behavior and seems way better than having a dependency on `lodash`! +1